packages/cron/src/lib/email.js

```diff
@@ -79,4 +79,42 @@ export class EmailService {
       messageId
     })
   }
+
+  /**
+   * Send an email to admin with a list of all users that have exceeded their storage quota
+   * @param {object} opts
+   * @param {Array<import('@web3-storage/db/db-client-types').UserStorageUsedOutput>} opts.users
+   * @param {string} opts.email
+   * @param {number} opts.secondsSinceLastSent
+   * @returns void
+   */
+  async sendAdminEmail ({
+    users,
+    email,
+    secondsSinceLastSent
+  }) {
+    const adminUser = await this.db.getUserByEmail('admin@web3.storage')
```

Comment:
Good point about the admin user, have added the user creation as a migration script so it doesn't get forgotten.

The `emailHasBeenSent` has been left as supporting just user IDs since we only have the one extra case for user by email address. Should be revisited in future.

Reply:
Good idea creating the migration 👌